content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -10,61 +10,48 @@ sidebar_current: "docs-enterprise2-api-workspaces"
 
 Workspaces represent running infrastructure managed by Terraform.
 
+## Create a Workspace
 
-
-
-## Create a Workspace with a VCS Repository
-
-This endpoint is used to create a new workspace which references an `oauth-token`, `linkable-repo-id` and `ingress-trigger-attributes` to configure the connection to VCS.
+This endpoint is used to create a new workspace either with or without a VCS configuration.
 
 | Method | Path           |
 | :----- | :------------- |
-| POST | /compound-workspaces |
+| POST | /organizations/:organization_username/workspaces |
 
+### Parameters
 
+- `:organization_username` (`string: <required>`) - Specifies the username or organization name under which to create the workspace. The organization must already exist in the system, and the user must have permissions to create new workspaces. This parameter is specified in the URL path.
 
-### Parameters
+### Create a Workspace Without a VCS Repository
 
-- `:organization` (`string: <required>`) - Specifies the username or organization name under which to create the workspace. The organization must already exist in the system, and the user must have permissions to create new workspaces. This parameter is specified in the URL path.
-- `oauth-token-id` (`string: <optional>`) - Specifies the VCS Connection (OAuth Conection + Token) to use as identified. This ID can be obtained from the [oauth-tokens](./oauth-tokens.html) endpoint.
-- `default-branch` (`boolean: true`) - specifies if the default branch should be used.
-- `ingress-submodules` (`boolean: false`) - Specifies whether submodules should be fetched when cloning the VCS repository.
-- `linkable-repo-id` (`string: <optional>`) - This is the reference to your VCS repository in the format :org/:repo where :org and :repo refer to the organization and repository in your VCS provider.
-- `name` (`string: <required>`) - Specifies the name of the workspace, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.
-- `working-directory` (`string:''`) - Specifies the directory that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository.
+If you supply nothing but a name, you can create a Workspace without configuring it against a VCS Repository.
```

Comment:
In sentence text, objects like "workspace" and "repository" shouldn't be capitalized as proper nouns.

Reply:
There should be a parameters list here that includes `name`, since it looks like that's required.